doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,122 @@ a single confidential command to let the redirection also apply for 'set -x'.
+```
+
+NOTE: See `zipl(8)` for all device types supported by the `--target` option
+      (e.g. DASD, SCSI, ...). If you want to use a tape device, see `zipl(8)`
+      to use the `--tape` option instead. Note that the `--tape` option with
+      ReaR is untested so contributions to this guide are welcome! Also, it is
+      not possible to use `zipl` with the z/VM reader device.
+
+IMPORTANT: The following method will overwrite the boot configuration on a device
+           which contains the given `--target` directory, in our case
+           `/dev/dasda`!
+
+Use the following command to make the device mounted at `/mnt/bootdisk` bootable
+with `zipl`:
+```
+# zipl --target=/mnt/bootdisk \
+       --parameters='root=/dev/ram0 ro' \
+       --image=/mnt/bootdisk/kernel-HOSTNAME \
+       --ramdisk=/mnt/bootdisk/initramfs-HOSTNAME.img
+Run /lib/s390-tools//zipl_helper.device-mapper /mnt/bootdisk/
+Building bootmap in '/mnt/bootdisk/'
+Adding IPL section
+  initial ramdisk...: /mnt/bootdisk/initramfs-ibm-z-523.img
+  kernel image......: /mnt/bootdisk/kernel-ibm-z-523
+  kernel parmline...: 'root=/dev/ram0 ro'
+  component address:
+    internal loader.: 0x0000a000-0x0000dfff
+    parameters......: 0x00009000-0x00009fff
+    kernel image....: 0x00010000-0x0076afff
+    parmline........: 0x0076c000-0x0076cfff
+    initial ramdisk.: 0x00780000-0x28708fff
+    environment blk.: 0x0077c000-0x0077cfff
+Preparing boot device for CCW- and LD-IPL: dasda.
+Done.
+```
+
+If you want to boot the device with ReaR recovery from a running Linux
+instance, you can use the `chreipl` tool to select a new default boot device
+and reboot:
+```
+# chreipl /mnt/bootdisk
+Re-IPL type: ccw
+Device:      0.0.0120
+Loadparm:    ""
+Bootparms:   ""
+clear:       0
+# reboot
+...
+```
+
+If you do not have a running Linux instance at hand, you can force the boot
+from the given device using the z/VM Control Program (CP) directly, e.g. through
+a terminal connection.
+
+The following example only considers DASD disk devices. Consult the official
+z/VM CP Manual for booting from other devices (e.g. FCP attached SCSI) or for
+additional details in general.
+
+Use the following command to boot from a DASD boot device through z/VM CP.
+Do not forget to include the `#` prefix which marks commands for z/VM CP!
+```
+#CP IPL vdev
+```
+
+The virtual device number (`vdev`) can be either obtained from the `Bus-ID`
+column using the `lsdasd -s` command from a running Linux instance (`0120`):
+```
+# lsdasd -s
+Bus-ID    Status    Name      Device  Type         BlkSz  Size      Blocks
+================================================================================
+0120      active    dasda     94:0    ECKD         4096   39128MB   10017000
+```
+or directly from z/VM CP with the following query:
+```
+#CP QUERY VIRTUAL DASD
+00:
+00: CP QUERY VIRTUAL DASD
+00: DASD 0120 ON DASD  2213 R/W 0X0120 SUBCHANNEL = 0004
+00: DASD 0121 ON DASD  201A R/W 0X0121 SUBCHANNEL = 0005
```

Comment:
I have an understanding question here:
From above I see that '0120' is "the right one"
but I do not understand how to see that in the
output of the `#CP QUERY VIRTUAL DASD` command,
i.e. why '0121' is not "the right one"
in particular compared to the output of
the `lsdasd -s` command that shows only one disk
so that this one disk must be "the right one".

Reply:
Yes, I also believe that there is no reliable way to tell which device is the right from CP.  (On the other hand, nothing stops you to just try `IPL` on all of them one by one).

By the way, it is possible to explicitly ignore devices when Linux boots which is the reason why CP may list more DASD devices than `lsdasd`.  See:
* `cioignore(8)` -  https://www.ibm.com/docs/en/linux-on-z?topic=commands-cio-ignore
* `cio-ignore=` kernel cmdline parameter - https://www.ibm.com/docs/en/linux-on-systems?topic=parameters-cio-ignore

Reply:
There are actually two ways of ignoring devices: one is to ignore them completely by the kernel (`cio_ignore` does that), and marking them offline, which will avoid creating /dev/dasd* device nodes for them and will show them only in the `lsdasd --offline` output.
CP and CMS will usually see more devices than Linux, because some of them are used by CMS and showing them as device nodes under Linux is usually not helpful.